README-pt-BR.md

```diff
@@ -0,0 +1,347 @@
+O *The Turing Way* é um manual para apoiar os alunos, seus supervisores, financiadores e editores de periódicos a garantir que a ciência de dados reproduzível seja "fácil demais para não ser feita", mesmo para pessoas que nunca trabalharam dessa forma antes.
+Ele inclui o material de treinamento sobre controle de versão, teste de análise e comunicação aberta e transparente com usuários futuros, além de estudos de caso e workshops do Instituto Turing(*Turing Institute*).  
+Este projeto é desenvolvido abertamente e todas e quaisquer perguntas, comentários e recomendações são bem-vindas em nosso repositório do GitHub: <https://github.com/alan-turing-institute/the-turing-way>.
+
+### O time
+
+Esta é uma parte do trabalho de planejamento da equipe do projeto no Turing Institute.  
+Para mais informações sobre como nos contactar, consulte o [documento com as formas de trabalho](ways_of_working.md).
+
+![Foto do time](book/website/figures/TuringWayTeam.jpg)
+
+### Como contribuir
+
+:construction: Este repositório é sempre um trabalho em andamento e **todos** são incentivados a nos ajudar a construir algo que seja útil para muitos. :construction:
+
+Todos são convidados a seguir nosso [código de conduta](CODE_OF_CONDUCT.md) e a checar nossas [diretrizes de contribuição](CONTRIBUTING.md) para obter mais informações sobre como começar.
+
+Se você não está familiarizado ou confiante em contribuir com o GitHub, também pode contribuir com um estudo de caso e com suas dicas e truques por meio de nosso [formulário Google](https://goo.gl/forms/akFqZEIy2kxAjfZW2).
+
+### Citando o _The Turing Way_
+
+Você pode fazer referência ao _The Turing Way_ através do arquivo Zenodo do projeto usando DOI: [10.5281/zenodo.3233853](https://doi.org/10.5281/zenodo.3233853).
+Os DOIs nos permitem arquivar o repositório e são realmente valiosos para garantir que o trabalho seja rastreado em publicações acadêmicas.
+
+A citação consistirá em algo parecido com:
+
+> The Turing Way Community, Becky Arnold, Louise Bowler, Sarah Gibson, Patricia Herterich, Rosie Higman, … Kirstie Whitaker. (2019, March 25). The Turing Way: A Handbook for Reproducible Data Science (Version v0.0.4). Zenodo. <http://doi.org/10.5281/zenodo.3233986>
+
+Você também pode compartilhar um URL mais legível que direciona a uma página do livro, por exemplo: <https://the-turing-way.netlify.com/reproducibility/03/definitions.html>, mas esteja ciente de que o projeto está em desenvolvimento, e dessa forma, esses links podem mudar com o tempo.
+Você pode querer incluir um link arquivado na web, como: <https://web.archive.org/web/20191030093753/https://the-turing-way.netlify.com/reproducibility/03/definitions.html> Assim você se certifica de não acabar com links quebrados por aí!
+
+Nós realmente apreciamos qualquer referência que você fizer ao projeto The Turing Way e esperamos que seja lhes seja útil.
```

Comment:
Nós realmente apreciamos qualquer referência que você fizer ao projeto The Turing Way e esperamos que lhe seja útil.

Reply:
Done!